### ISSUE TITLE
[Question] is project development discontinued?

### ISSUE THREAD

**Issue by yellowmegaman** [asteris-llc/converge#640]
Hi there!

Awesome tool, found in packer.io docs.

Just noticed that last release was just a couple of days more than a year ago.
That's extremely sad. 

Actually going to use it anyway, need lightweight replacement to ansible, for some infra provisioning ( just some binaries and file contents).

Thanks a lot in advance for reply!

**Comment by mstevens-sc** [comment]
I agree with @yellowmegaman. I think that there’s a lot of potential here for keeping configuration in HCL along with other tools like Terraform that utilize it as well. I hope someone is able to keep this going (I’m not a Go dev, unfortunately).

**Comment by stevendborrelli** [comment]
Development is not discontinued, but as a consulting company we currently don't have the cycles to do updates. The core use cases of converge (an easy way to create graph-dependent workflows) are something we still are very interested in pursuing.

We've been discussing features internally, and I hopeful we will soon have a project to assist in development.

**Comment by ahane** [comment]
could you give a 2019 update on this project? :)

**Comment by stevendborrelli** [comment]
We won't be providing updates to this project, as most of our focus is on Kubernetes. We don't have a use case that we can use to justify allocating resources to the tool. :(

If someone is willing to take over maintenance, I would be happy to assist.

**Comment by the-maldridge** [comment]
Given that this project is now abandon-ware, I'd highly recommend adding a notice to the top of the readme.

**Comment by marco-m** [comment]
Hello @stevendborrelli, just found this project following Packer documentation. I could have written the same text as @yellowmegaman wrote above :-).

Just wanted to tell to the dev team that I am impressed. In the configuration management space, I come from SaltStack and Ansible and suffered enough. Sometimes one cannot get away with a Dockerfile, not everything is immutable. When that happens, Converge is simply so modern, kudos for the design and the tech choices. Too bad that you don't have the time any more to maintain, although I understand perfectly.
